_archetypes/errai-archetypes

```diff
@@ -1 +1 @@
-Subproject commit 79fc168d19b86fd9bb5e7e7f0b838e14ee04e0c9
+Subproject commit b7669ad1c92554da3b82a8f205dd72f96e145d38
```

Comment:
You need to revert this change. It looks to be breaking the build and it shouldn't be in here.